lib/index.ts

```diff
@@ -22,3 +22,4 @@ export {
   Func, AmqpResponseStatusCode, isAmqpError, ConnectionStringParseOptions, delay, messageHeader,
   messageProperties, parseConnectionString, ParsedOutput
 } from "./util/utils";
+export { Typed } from "rhea/typings/types";
```

Comment:
Rather than exporting Typed from the file path for types, it would be better to export `Typed` from `rhea`.

```ts
export { types as Types, Typed } from "./types";
```
over [here](https://github.com/amqp/rhea/blob/master/typings/index.d.ts#L15) and then re-export it in this index.ts file over [here](https://github.com/amqp/rhea-promise/pull/36/files#diff-13b5b151431c7e7a17f273559ed212d5R10).

Reply:
Got it - submitted PR for this at - https://github.com/amqp/rhea/pull/213